engines/misc/src/main/java/org/alfresco/transform/misc/transformers/TextToPdfContentTransformer.java

```diff
@@ -231,22 +277,35 @@ static PDType1Font getStandardFont(String name)
         }
         //duplicating until here
 
+        private String fontName = null;
+        private boolean fontChanged = false;
+        private String defaultFont = null;
+
         // The following code is based on the code in TextToPDF with the addition of
         // checks for page limits.
         // The calling code must close the PDDocument once finished with it.
-        public PDDocument createPDFFromText(Reader text, int pageLimit)
+        public PDDocument createPDFFromText(Reader text, int pageLimit, String pdfFontName, Integer pdfFontSize)
             throws IOException
         {
             PDDocument doc = null;
             int pageCount = 0;
             try
             {
+                doc = new PDDocument();
+
+                final PDFont font = getFont(doc, pdfFontName);
+                final int fontSize = pdfFontSize != null ? pdfFontSize : getFontSize();
+
+                fontName = font.getName();
+
+                logger.debug("Going to use font " + fontName + " with size " + fontSize);

```

Comment:
We can use the [SLF4J placeholders](https://www.slf4j.org/manual.html#typical_usage) to format strings rather than concatenating them.